pages/movies/[id].js

```diff
@@ -0,0 +1,65 @@
+// import DetailList from "../../components/DetailList";
+import { useRouter } from "next/router";
+import styled from "styled-components";
+import Image from "next/image";
+import Link from "next/link";
+
+export default function MovieDetailPage({ movies }) {
+  console.log(movies);
+
+  const { query } = useRouter();
+  const { id } = query;
+
+  function getMovieById(id) {
+    return movies.find((movie) => movie.id === id);
+  }
+
+  const movieDetail = getMovieById(id);
+  //   console.log(movieDetail);
+  if (!movieDetail) return <h2>loading..</h2>;
+
+  //   const splittingGenre = movieDetail.genre.split(",");
+  //   console.log(splittingGenre);
+
+  return (
+    <>
```

Comment:
```suggestion
    <main>
```
hier könntest du einen main tag verwenden